src/components/Placeholder.js

```diff
@@ -0,0 +1,24 @@
+export default function Placeholder(props) {
+  const {
+    text,
+    icon: Icon
+  } = props;
+
+  return (
+    <div class="bio-properties-panel open">
+      <section class="bio-properties-panel-placeholder">
+        { Icon && <Icon class="bio-properties-panel-placeholder-icon" /> }
+        <p class="bio-properties-panel-placeholder-text">{ text }</p>
+      </section>
+    </div>
+  );
+}
+
+const noop = () => null;
+
+export const NOOP_PROVIDER = {
+  getEmptyText: noop,
```

Comment:
Any reason we don simplify this to `placeholder = { text, icon? }` and make a provider offer `placeholderProvider.empty()` and `placeholderProvider.multiple()`. For me this looks like a duplication to have `get*Text` and `get*Icon` in the API.

Reply:
Good thing, let's try to simplify this 👍

Reply:
I updated it accordingly, the placeholder has now to provide `.empty()` and `.multiple()` 👍